src/typescript/06-types/types.md

```diff
@@ -185,12 +185,12 @@ type Intersection = Union1 & Union2;
 ```ts
 type Object1 = {
   name: string;
-  age: number;
+  age: number; // <--
 };
 
 type Object2 = {
   description: string;
-  age: string;
+  age: string; // <--
```

Comment:
I think the `@highlight` directive would be useful here, and/or the comment should explain what is being pointed out?